### PR TITLE
Photos monday

### PR DESCRIPTION
**Pull request by jwnelen**
## Changed
- Added the new photos of monday.
- Fixed the optimize-image method
- Renamed the image folders

## Still want to improve
- Want to change the images such that when you click on it, you see the full sized image in a new tab.

Other then that, nothing :)

### CODE REVIEW

**Comment by TimvdLippe** [comment]
> Want to change the images such that when you click on it, you see the full sized image in a new tab.

That would mean you also need to upload the original images. Then you can wrap the images in a `<a href="<url-but-then-to-images-instead-of-images-optimized>" target="_blank">`

File: index.html [hunk 1]
```diff
@@ -54,44 +54,77 @@ <h3>Photos</h3>
...
-                <div> <img data-src="images-optimized/pre-pre-dies/DSC_0115_result.png" />Marleen</div>
-                <div> <img data-src="images-optimized/pre-pre-dies/DSC_0116_result.png" />Marleen, Vera and Louise</div>
-                <div> <img data-src="images-optimized/pre-pre-dies/DSC_0121_result.png" />Yeaass PIE</div>
-                <div> <img data-src="images-optimized/pre-pre-dies/DSC_0124_result.png" />Daniel</div>
-                <div> <img data-src="images-optimized/pre-pre-dies/DSC_0126_result.png" />Thomas and Karim</div>
+                <div> <img data-src="images-optimized/prepredies/DSC_0115_result.png" />Marleen</div>
+                <div> <img data-src="./images-optimized/prepredies/DSC_0116_result.png" />Marleen, Vera and Louise</div>
+                <div> <img data-src="./images-optimized/prepredies/DSC_0121_result.png" />Yeaass PIE</div>
+                <div> <img data-src="./images-optimized/prepredies/DSC_0124_result.png" />Daniel</div>
+                <div> <img data-src="./images-optimized/prepredies/DSC_0126_result.png" />Thomas and Karim</div>
```
Comment (TimvdLippe) [inline index.html]: This breaks the loading logic of images, you need to update the `id` as well.

Comment (TimvdLippe) [reply index.html]: Also the `./` is probably not necessary

File: style.css [hunk 2]
```diff
@@ -239,6 +239,18 @@ button:active {
   padding-top: 200px
 }
 
+#monday img{
+  height: 300px;
```
Comment (TimvdLippe) [inline style.css]: Why would the monday images be styled differently?

**Review by TimvdLippe — CHANGES_REQUESTED** [review]

File: style.css (hunk 2 above)
Comment (jeroentjuuhh) [reply style.css]: Most images are vertical whereas the images of the pre-dies are horizontal. This vertical images where too big with the same width as the horizontal photos

Comment (TimvdLippe) [reply style.css]: Okay, seems fine. We probably want to later change the selector to `.vertical-day` or something, but that's in another PR

Comment (jeroentjuuhh) [reply style.css]: Yeah, agree!

**Review by TimvdLippe — APPROVED** [review]

**Pull request merged**